src/Storage/Storage.Management.Test/ScenarioTests/StorageTestRunner.cs

```diff
@@ -31,6 +32,9 @@ protected StorageTestRunner(ITestOutputHelper output)
                     @"Common.ps1",
                     @"../AzureRM.Resources.ps1"
                 })
+                .WithRecordMatcher((ignoreResourcesClient, resourceProviders, userAgentsToIgnore) =>
```

Comment:
What I mean is the record matcher arguments instead of the matcher itself. In the test controller, there are arguments for the matcher, also need to pass in these arguments in the test runner. The WithRecordMatcher invocation here can be removed as it only instantiated the default matcher.

Reply:
Sorry, I didn't notice "arguments".